src/data/faq_de.json

```diff
@@ -1887,6 +1887,67 @@
+                            "Zusätzlich reagieren schon allein wegen unterschiedlicher technischer Ausstattung (Hardware) und Systemsoftware unterschiedlich bzgl. Sendeleistung/-signalstärke und Empfangssensibilität. Auch zwei nebeneinander liegende Geräte empfangen daher nicht zwangsläufig dieselben Diagnoseschlüssel zur selben Zeit mit identischen Parametern.",
+                            "Siehe auch FAQ: <ul><li><a href='#risk_encounter_not_realtime' target='_blank'>Warum erhalte ich erst nach mehreren Tagen nach der Risikobegegnung eine Risikowarnung?</a></li><li><a href='#risk_encounter_no_warning' target='_blank'>Warum erhalte ich keine Risikowarnung, obwohl ich Kontakt zu einer Person hatte, die eine Risikobegegnung in ihrer App angezeigt bekommt?</a></li><li><a href='#pos_test_no_risk_encounter' target='_blank'>Ich wurde positiv getestet. Warum erhält eine Person im gleichen Haushalt keine Risikowarnung?</a></li><li><a href='#risk_encounter_doubted' target='_blank'>Warum erhalte ich eine Risikowarnung für ein Datum, an dem ich keine Kontakte hatte / nicht das Haus verlassen habe?</a></li><li><a href='#risk_encounter_place' target='_blank'>Werden zu einer Risiko-Begegnung Personen und/oder Orte zugeordnet?</a></li></ul>"
+                        ]
+                    },
+                    {
+                        "title": "Ich wurde positiv getestet. Warum erhält eine Person im gleichen Haushalt keine Risikowarnung?",
+                        "anchor": "pos_test_no_risk_encounter",
+                        "active": false,
+                        "textblock": [
+                            "Um Warnungen zu erhalten, gibt es gewisse Voraussetzungen, die erfüllt sein müssen:<ul><li>Nach dem Erhalt des positiven Testergebnis muss dieses freiwillig vom Nutzer über die Corona-Warn-App geteilt werden. Falls das Testergebnis nicht über die App ermittelt werden konnte, kann das positive PCR-Testergebnis über eine TAN eingetragen und geteilt werden.</li><li>Die App muss bereits <b>vor</b> dem Teilen des positiven Testergebnis über die Corona-Warn-App auf beiden Geräten installiert gewesen sein, da ansonsten keine Begegnungen aufgezeichnet wurden.</li><li>Die beiden Geräte müssen sich <b>vor</b> dem Teilen des Testergebnis nahe und lange genug beieinander befunden haben. Begegnungen <b>nach</b> dem Teilen eines positiven Befundes werden natürlich <b>nicht</b> gewarnt.</li><li>Auch der beim Teilen des Testergebnis angegebene Zeitraum für das Auftreten der ersten Symptome, sowie die Art bzw. Intensität der Symptome kann bei der Risikoberechnung eine Rolle spielen. Es kommt also auch darauf, an welchen Tagen vor dem Teilen des Testergebnisses die Geräte der Personen die Begegnungen aufgezeichnet haben. In einigen Fällen kann dies dazu führen, dass keine Warnung versendet wird.</li></ul>",

```

Comment:
"Die App muss bereits <b>vor</b> dem Teilen des positiven Testergebnis über die Corona-Warn-App auf beiden Geräten installiert gewesen sein, da ansonsten keine Begegnungen aufgezeichnet wurden."

This is necessary, but not sufficient! Exposure Logging and Bluetooth have to be active.

Reply:
"installiert **und aktiv**" would maybe be better.

Reply:
Changed it to "installiert und aktiv" in https://github.com/corona-warn-app/cwa-website/pull/2529/commits/e25bfbeb50abbc2054e6fed0564c3cd2e725c5c9, thanks!